docs/deploy.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
```

Comment:
Maybe add an assertion that cwd is the docs directory

Reply:
I used explicit paths relative to `scriptdir=$(dirname ${BASH_SOURCE[0]})`.